model-optimizer/mo/main.py

```diff
@@ -147,6 +148,19 @@ def prepare_ir(argv: argparse.Namespace):
     if not argv.silent:
         print_argv(argv, is_caffe, is_tf, is_mxnet, is_kaldi, is_onnx, argv.model_name)
 
+    # this try-except is just an additional reinsurance to be sure that
+    # IE dependency search doesn't break MO pipeline
+    try:
+        if not find_ie_version(silent=argv.silent) and not argv.silent:
+            print("[ WARNING ] No Inference Engine python was found. At this moment Inference Engine dependency is not mandatory but in future it will be required.")
```

Comment:
```suggestion
            print("[ WARNING ] Could not find the Inference Engine Python. At this moment, the Inference Engine dependency is not mandatory, but it will be required later.")
```
Probably need to specify what is later

Reply:
Done.